knotx-core/knotx-repository/knotx-repository-filesystem/src/main/java/com/cognifide/knotx/repository/FilesystemRepositoryVerticle.java

```diff
@@ -84,7 +84,7 @@ public void start() throws Exception {
 
     LOGGER.trace("Fetching file `{}` from local repository.", localFilePath);
 
-    return fileSystem.openObservable(localFilePath, new OpenOptions())
+    return fileSystem.openObservable(localFilePath, new OpenOptions().setCreate(false))
```

Comment:
setWrite(false) is also required as it's true by default.
Instead of specifing it inline, please create final field of the class with OpenOptions set to write=false and create=false.